src/sagemaker/session.py

```diff
@@ -4363,9 +4363,9 @@ def get_create_model_package_request(
     if validation_specification:
         request_dict["ValidationSpecification"] = validation_specification
     if containers is not None:
-        if not all([content_types, response_types, inference_instances, transform_instances]):
+        if not all([content_types, response_types]):
             raise ValueError(
-                "content_types, response_types, inference_inferences and transform_instances "
+                "content_types and response_types "
                 "must be provided if containers is present."
             )
```

Comment:
This thrown error was added by sreedes@. Did you check with her to see if she agree on this?

Reply:
Discussed the same with Saumitra and got approval for this change